nntrainer/layers/reshape_layer.h

```diff
@@ -78,6 +78,17 @@ class ReshapeLayer : public Layer {
    */
   bool supportInPlace() const override { return true; }
 
+  /**
+   * @brief Initialize the in-place type of the layer
+   * @return InPlaceType
+   */
+  InPlaceType initializeInPlaceType() override {
```

Comment:
Simple question!
It seems flatten_layer, which inherits ReshapeLayer, overrides the initializeInPlaceType() with the same code.
If you think it is always same, what about defining this function as `final` instead of `override` to reduce code replication?

Reply:
@EunjuYang  That's a good point! I set that function of the reshape layer to be `final` instead of `override`, and removed that function from the flatten layer. Thank you.